botorch/models/model.py

```diff
@@ -246,16 +253,24 @@ def train(self, mode: bool = True) -> Model:
 
 
 class ModelList(Model):
-    r"""Container for a list of models."""
+    r"""A multi-output Model represented by a list of independent models.
+
+    A multi-output Model represented by a list of independent models. Any
```

Comment:
Repeats the same sentence
```suggestion
    All
```